transport/socks5/socks5.go

```diff
@@ -109,6 +110,6 @@ func appendSOCKS5Address(b []byte, address string) ([]byte, error) {
 		b = append(b, byte(len(host)))
 		b = append(b, host...)
 	}
-	b = append(b, byte(portNum>>8), byte(portNum))
+	b = binary.BigEndian.AppendUint16(b, uint16(portNum))
```

Comment:
This is to resolve the CodeQL error: https://github.com/Jigsaw-Code/outline-sdk/pull/133/checks?check_run_id=18912663629

(Although the standard library `AppendUint16` is using the same code to append the data)